### PR TITLE
AAP-18761 convert/create modules for the gateway dashboard

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR converts the draft of the gateway dashboard content into asciidoc and adds the new modules to the modules/platform directory in the repo.

See draft here for reference:

https://docs.google.com/document/d/1traqmk9K88VLGaLP8xPpnvtdLTx4uP0ql8TVKQhi6gM/edit?usp=drive_link

### CODE REVIEW

File: downstream/modules/platform/con-gw-dash-components.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="gw-dash-components_{context}"]
+
+= {PlatformNameShort} dashboard components
+
+Quick starts:: You can learn about Ansible automation functions with guided tutorials called quick starts. In the dashboard, you can access quick starts by selecting a quick start card. From the panel displayed, click Start and complete the onscreen instructions. You can also filter quick starts by keyword and status.
```
Comment (ianf77) [inline downstream/modules/platform/con-gw-dash-components.adoc:7]: should it be click btn:[Start] ?

File: downstream/modules/platform/con-gw-dash-components.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Resource status:: Indicates the status of your hosts, projects and inventories. The status indicator links to your configured hosts, projects and inventories where you can search, filter, add and modify these resources. 
+Job Activity:: You can view a summary of your current job status, filter the job status within a period of time or by job type, or click Go to Jobs to view a complete list of jobs that are currently available.
```
Comment (ianf77) [inline downstream/modules/platform/con-gw-dash-components.adoc:9]: btn:[Go to Jobs] ?

**Review by ianf77 — COMMENTED** [review]
A couple of comments

File: downstream/modules/platform/con-gw-dash-components.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-gw-dash-components.adoc]: Good catch; thank you!

File: downstream/modules/platform/con-gw-dash-components.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-gw-dash-components.adoc]: These are actually links on a modal, so I've bolded them according to UI guidelines. Great catch!

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**